spec/support/path.rb

```diff
@@ -108,10 +108,6 @@ def lib_path(*args)
       tmp("libs", *args)
     end
 
-    def bundler_path
```

Comment:
I think we restore `bundler_path` and use it instead like `require "../../lib/bundler"`. Because of the directory structure of the ruby core repository for spec files are different from this repository.

Reply:
The reason I removed it is because it was duplicated with the `lib` helper method. But I agree with your suggestion, so I will switch to it :+1:. Thanks!

Reply:
I moved the `lib` helper to `spec/support/path.rb`, since it had the exact same implementation of this method. Now the `lib` helper is used everywhere.